File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/JobApplicationApprovalResponseDto.java
package com.ungs.docsys.dtos;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class JobApplicationApprovalResponseDto {
    private Long id;
    private JobApplicationResponseDto jobApplication;
    private Boolean approved;
    private String reason;
    private AppUserResponseDto appUser;
    private LocalDateTime createdDate;
    private LocalDateTime updatedDate;
}

File: ungs-docsys-bff/src/modules/job-applications/job-application-approvals/dtos/job-application-approval-response.dto.ts
import { AppUserExtendedResponseDto } from "src/modules/login/app-user/dtos/app-user-extended-response.dto";
import { JobApplicationResponseDto } from "../../dtos/job-application-response.dto";

export class JobApplicationApprovalResponseDto {
    id: number;
    jobApplication: JobApplicationResponseDto;
    approved: boolean;
    reason: string;
    appUser: AppUserExtendedResponseDto;
    createdDate: Date;
    updatedDate: Date;
}